canonical-websites/www.ubuntu.com
# Move release EOL page to /about/release-cycle and update copy

## Done

- Moved /info/release-end-of-life page to /about/release-cycle
- Updated copy according to the [copydoc](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit#)
- Added redirects to new page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/about/release-cycle
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that the page content matches the [copydoc](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit#)
- **Note that the /about/kernel page has not been finalised yet, so for now the kernel section has a link to wiki.ubuntu.com/Kernel**
- Go to http://0.0.0.0:8001/info/release-end-of-life and check that it redirects to http://0.0.0.0:8001/about/release-cycle


## Issue / Card

Fixes #3842
